pkg/cli/clisqlshell/sql.go

```diff
@@ -276,7 +276,7 @@ func (c *cliState) printCliHelp() {
 	fmt.Fprintf(c.iCtx.stdout, helpMessageFmt,
 		demoHelpStr,
 		docs.URL("sql-statements.html"),
-		docs.URL("use-the-built-in-sql-client.html"),
+		docs.URL("cockroach-sql.html"),
```

Comment:
Not for this PR, but the link without the extension is preferred. The website automatically redirects from the page with `.html` to the one without.